docs/build/linux-ubuntu.md

```diff
@@ -49,6 +49,10 @@ sudo apt install devscripts equivs
 sudo mk-build-deps --install linux/debian/control
 ```
 
+### Node.js
+
+Node.js v18 is required for the core build.
```

Comment:
```suggestion
Node.js v18 is required for Core build, Web tests, and Developer command line tools.
```

I think I'd like to spend a day on the dev docs at some point to get them consistent once more -- the Win/mac docs also need this info...